desktop/extensions-sdk/extensions/index.md

```diff
@@ -4,40 +4,65 @@ description: General steps in how to publish an extension
 keywords: Docker, Extensions, sdk, publish
 ---
 
-To publish the extension, upload the Docker image to [DockerHub](https://hub.docker.com/).
+This section provides a how-to guide, general information on publishing your extension to the Extensions Marketplace.
 
-Tag the previous image to prepend the account owner at the beginning of the image name:
+For information on how Docker Extensions are packaged and distributed, see [Packaging, distribution and API dependencies](DISTRIBUTION.md)
 
-`docker tag <name-of-your-extension> owner/<name-of-your-extension>`
+## How to publish your extension 
 
-Push the image to Docker Hub:
+To publish the extension you need to upload the Docker image to [DockerHub](https://hub.docker.com/):
 
-`docker push owner/<name-of-your-extension>`
+1. Tag the previous image to prepend the account owner at the beginning of the image name:
 
-> Note
-> 
-> For Docker Extensions images to be listed in Docker Desktop, they must be approved by Docker and the tags must follow semantic versioning, e.g: `0.0.1`.
-> 
-> See [distribution and new releases](DISTRIBUTION.md#distribution-and-new-releases) for more information.
-> 
-> See [semver.org](https://semver.org/) to learn more about semantic versioning.
-> 
+    `docker tag <name-of-your-extension> owner/<name-of-your-extension>`
```

Comment:
````suggestion
    ```console
    $ docker tag <name-of-your-extension> owner/<name-of-your-extension>
    ```
````